assets/js/components/notifications/notification.js

```diff
@@ -55,14 +60,24 @@ class Notification extends Component {
 		}
 	}
 
-	handleDismiss( e ) {
-		const { isClosed } = this.state;
-		const card = this.cardRef.current;
-
+	async handleDismiss( e ) {
+		e.persist();
```

Comment:
What's this for?

Reply:
Since these events are actually [`SyntheticEvents`](https://reactjs.org/docs/events.html#overview) from React, we need to persist them using this method to ensure they are not disposed of to make them available to any subsequent callbacks. We don't actually use the events right now, but if we're going to pass it, it should be persisted.